x-pack/docs/en/rest-api/security/create-api-keys.asciidoc

```diff
@@ -57,6 +61,12 @@ thereby limiting the access scope for API keys.
 The structure of role descriptor is the same as the request for create role API.
 For more details, see <<security-api-put-role, create or update roles API>>.
 
+NOTE: Due to the way in which this permission intersection is calculated, it is not
+possible to create an API key that is a child of another API key, unless the derived
+key is created without any privileges. In this case, you must explicitly specify a
+role descriptor with no privileges, and the derived API key can be used for
+authentication only, but will not have access to call {es} APIs.
```

Comment:
I think "but" should be removed here. Should "access" also be changed to "authority"?
```suggestion
authentication only; it will not have authority to call {es} APIs.
```

Reply:
> Without complicating things too much by explaining all the details, maybe we could just drop the `only` bit.

I've made that change in my suggestion, thanks!